File: HelperJobby/DTOs/Organization/OrganizationDTO.cs
using HelperJobby.DTOs.Account;

namespace HelperJobby.DTOs.Organization;

public class OrganizationDTO
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string PhoneNumber { get; set; }
    public int? NumberOfEmployees { get; set; }
    public int OrganizationOwnerId { get; set; }
    public List<EmployerDTO> EmployeeAccounts { get; set; }
    public List<CreateOrganizationEmployeeEmailDTO> EmployeeEmails { get; set; }
}

File: HelperJobby/client-app/src/DTOs/OrganizationDTOs/OrganizationDTO.ts
import {EmployerAccountDTO} from "../AccountDTOs/EmployerAccountDTO";
import {CreateOrganizationEmployeeEmailDTO} from "./CreateOrganizationEmployeeEmailDTO";

export interface OrganizationDTO {
    id: number;
    name: string;
    phoneNumber: string;
    numberOfEmployees: number;
    organizationOwnerId: number;
    employeeAccounts: EmployerAccountDTO[];
    employeeEmails: CreateOrganizationEmployeeEmailDTO[];
}